src/bin/scheduler/setup/rcc.rs

```diff
@@ -122,6 +118,44 @@ impl SetupStep for StepFilePermissions {
     }
 }
 
+struct StepRCCCommand {
+    config: GlobalConfig,
```

Comment:
Adding `GlobalConfing` as a field is not a good idea. It makes constructing `StepRCCCommand` far more complicated to construct. This is a problem for testing.